src/preprocessing/passes/sygus_abduct.h

```diff
@@ -56,6 +56,18 @@ class SygusAbduct : public PreprocessingPass
  public:
   SygusAbduct(PreprocessingPassContext* preprocContext);
 
+  /**
+   * Returns the sygus conjecture corresponding to the abduction problem for
+   * input problem given by asserts, and axioms given by axioms. Note that
+   * axioms is expected to be a subset of asserts.
```

Comment:
Could you clarify what `axioms` and `asserts` are? Maybe in terms of traditional A and B such that A does not imply B? In particular, why should `axioms` be a subset of `asserts`?